fieldsolver/gridGlue.cpp

```diff
@@ -63,7 +69,8 @@ template <typename T, int stencil> void computeCoupling(dccrg::Dccrg<SpatialCell
 						      (uint64_t)globalIndices[1],
 						      (uint64_t)globalIndices[2]}}; //cast to avoid warnings
 	CellID dccrgCell = mpiGrid.get_existing_cell(indices, 0, mpiGrid.mapping.get_maximum_refinement_level());
-	int process = mpiGrid.get_process(dccrgCell);
+  
+  int process = mpiGrid.get_process(dccrgCell);
```

Comment:
And let's wrap up this short review with a new complaint about weird indentations. ;)

But, still: please use 3-space indentation levels as we have throughout the code. Let's not mix that or it will become a pain later. Some of the files I poked at had 2-space indentation levels.

Reply:
Ok updating shortly!

Reply:
:)

Reply:
!